### ISSUE TITLE
No. of dots don't change when deleting

### ISSUE THREAD

**Issue by RichestYoo** [tommybuonomo/dotsindicator#139]
Hello.
I found something not intuitive as following:
    When deleting an image from ViewPager2, the number of dots doesn't change.

        [Before Delete] 
                 (Image)  (Image)  (Image)           (Image - Marked)  <-- I delete last one
        [After Delete]
                 (Image)  (Image)  (Image - Marked)  (Image - appear but not exists)

As a quick fix, I put removeDot() with index from 0 to 3, but all 0~3 led to indexOutOfBound.
What am I supposed to do?

**Comment by kacsogellert** [comment]
I ended up fixing by calling refreshDots on the callback of the submitList method

```
  adapter.submitList(it){
             dotsIndicator.refreshDots()
      }
```

**Comment by tommybuonomo** [comment]
Hello, thanks for your contribution !
This issue is now fixed in the new version `4.3`
Thanks 🔥
